### ISSUE TITLE
72 warnings about deprecated stuff from org.apache.http

### ISSUE THREAD

**Issue by YarochkinMichael** [OPCFoundation/UA-Java#153]
Hello,

I am compiling project in IntelliJ and have 72 warnings about deprecated objects from org.apache.http library.

It is any plans to solve it?

Mikl

**Comment by jouniaro** [comment]
Yeah, that's since the HTTP Core libraries have deprecated stuff that we use. We should solve that at some point but it hasn't been high priority yet. There are no known technical issues with using the deprecated interfaces, though.
